src/chat/InvalidNarrow.js

```diff
@@ -27,7 +27,7 @@ type Props = $ReadOnly<{|
 export default function InvalidNarrow(props: Props): Node {
   return (
     <View style={styles.container}>
-      <ZulipTextIntl style={styles.text} text="That conversation doesn't seem to exist." />
+      <ZulipTextIntl style={styles.text} text="That conversation doesn’t seem to exist." />
```

Comment:
We caught all the straight quotes in f8351f0a6, but it looks like
    one crept in after that. Add this check so it doesn't happen again.

So to really accomplish that, there's one key step which this PR doesn't yet take: causing this suite to run in CI.

What runs in CI is controlled by `.github/workflows/`. That config is kept quite simple, with the interesting logic almost all pushed inside `tools/test`. The key bit is:
```
    - name: Run tools/test
      run: TERM=dumb tools/test --all
```
So basically it runs `tools/test --all`.

And `--all` doesn't currently change the set of suites to run:
```
  --all       In the given suites, run all tests on all files. Same as
              --all-files --platform both.
```

So that line of CI config is effectively relying on the fact that currently all suites run by default.

---

The crude fix here would be to add to that line of CI config a list of all the suites.

A cleaner fix would be to let the CI config just express "give me all the suites, whatever they are". Probably a good version of that is to change the definition of `--all` like so:
```
  --all       In the given suites, run all tests on all files, equivalently
              to \`--all-files --platform both\`. If no list of suites was specified,
              run all suites.
```
That way a simple `tools/test --all` does exactly what it sounds like it should do, while a command like `tools/test --all lint` still works as desired.

(Effectively `--all` has also been relying on the fact that all suites are run by default, so that this distinction doesn't matter.)